internal/server/routes.go

```diff
@@ -134,6 +135,9 @@ func (s *Server) GenerateRoutes() Routes {
 
 	add(a, noAuthnWithOrg, http.MethodGet, "/.well-known/jwks.json", wellKnownJWKsRoute)
 
+	// routes only interesting to connectors
+	get(a, authn, "/api/credentials", a.ListCredentialRequests)
```

Comment:
I expect the UI to want to list credentials as well, so that a user can revoke their own credentials, or an admin can revoke all of a users credentials.

Reply:
I don't think so; these are ephemeral things that expire quickly.

Reply:
Ah true, if these are very short lived (~5min) then remove isn't important